R/epinowcast.R

```diff
@@ -38,7 +38,8 @@
 #' `variable`, `mean`, `sd` describing normal priors. Priors in the
 #' appropriate format are returned by [enw_reference()] as well as by
 #' other similar model specification functions. Priors in this data.frame
-#' replace the default priors specified by each model component.
+#' replace the default priors specified by each model component. See the
+#' package vignette for more details on how to specify `priors`.
```

Comment:
is there a more helpful link than this to enable people to find the documentation?